portfolio/src/main/webapp/index.html

```diff
@@ -3,18 +3,79 @@
   <head>
     <meta charset="UTF-8">
     <title>Ethan's Portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Titillium+Web:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <script src="https://kit.fontawesome.com/75dadd3f2b.js" crossorigin="anonymous"></script>
```

Comment:
HTML loads documents from top to bottom. Because of this, it's a common practice to add script tags just above your closing body tag. This avoids page render delays due to expensive js.

Reply:
Ah okay. The walk through had it up there but I'll move it down.